src/Phinx/Migration/AbstractMigration.php

```diff
@@ -44,6 +44,12 @@
  */
 abstract class AbstractMigration implements MigrationInterface
 {
+    const PHINX_PARAM_BOOL = \PDO::PARAM_BOOL;
+    const PHINX_PARAM_NULL = \PDO::PARAM_NULL;
+    const PHINX_PARAM_INT = \PDO::PARAM_INT;
+    const PHINX_PARAM_STR = \PDO::PARAM_STR;
+    const PHINX_PARAM_LOB = \PDO::PARAM_LOB;
```

Comment:
I'm all for aliasing the `PDO` params to something local-to-the-class. `PDO` is a dependency, to which direct references should be discouraged.